_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -93,6 +93,8 @@ configure.args \
                     -DCMAKE_MODULE_PATH=${cmake_share_module_dir} \
                     -DCMAKE_FIND_FRAMEWORK=LAST \
                     -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+                    -DCMAKE_C_COMPILER=${configure.cc} \
+                    -DCMAKE_CXX_COMPILER=${configure.cxx} \
                     -Wno-dev
```

Comment:
FWIW I think they belong up here, but that will require switching to `default configure.args` so that subsequent changes to the compiler are picked up.